rfd/0003-deduplication.md

```diff
@@ -0,0 +1,214 @@
+# Deduplication of boefjes between organization
+
+The goal of deduplication of boefjes is to avoid doing multiple scans in similar
+organizations for the same assets. When a large number of organizations have the
+same asset this would prevent scanning the asset too many times and save
+resources.
+
+## Functional requirements
+
+Deduplication can only be done when the input of the boefje is exactly the same.
+This includes all the boefje settings. A few boefjes need to have the
+organization, so those boefjes can't be duplicated.
+
+When doing deduplication between organizations it would be possible for a user
+to see that a boefje task for a certain OOI was already done for an organization
+in the same KAT install earlier while the user might not have access to this
+organization. This might be a problem for certain usage of OpenKAT, so
+deduplication should be a setting that can be turned off.
+
```

Comment:
We should mention that we can only use previous scan raw files *if* they are not older than the requested interval in the requesting organization.